package.json

```diff
@@ -19,6 +19,7 @@
     "debug": "^2.2.0",
     "dotenv": "^2.0.0",
     "express": "^4.14.0",
+    "fuse.js": "^2.4.1",
```

Comment:
isn't it strange to use fussy search on client side when we have mongo already doing it for us?